client/src/components/PublicPages/AuditPlanner/AuditPlanCard.tsx

```diff
@@ -114,18 +125,18 @@ const AuditPlanCard: React.FC<IProps> = ({ disabled, electionResults }) => {
 
         <Section>
           <SubHeading>Risk Limit</SubHeading>
-          <Slider
+          <RiskLimitSlider
             disabled={disabled}
             labelRenderer={value => (
               <span
                 // A hack to display the listed values on both the slider axis and the slider label,
                 // and all other values on only the slider label. More recent versions of the
                 // Blueprint slider actually support this differentiation
-                style={{
-                  color: [0, 5, 10, 15, 20].includes(value)
-                    ? undefined
-                    : 'transparent',
-                }}
+                className={
+                  ![0, 5, 10, 15, 20].includes(value)
+                    ? HIDDEN_LABEL_CLASS_NAME
+                    : undefined
+                }
```

Comment:
Here's the issue I'm fixing here (note the streaks of white across the axis labels):

<img width="765" alt="issue" src="https://user-images.githubusercontent.com/12616928/193363579-99773246-e58d-4776-9351-c50bd27d0ace.png">